lib/MemoryModel/MemModel.cpp

```diff
@@ -189,7 +189,7 @@ bool SymbolTableInfo::computeGepOffset(const llvm::User *V, LocationSet& ls) {
         // Handling array types, skipe array handling here
         // We treat whole array as one, then we can distinguish different field of an array of struct
         // e.g. s[1].f1 is differet from s[0].f2
-        if(isa<ArrayType>(*gi))
+        if(isa<ArrayType>(gi.getIndexedType()))
```

Comment:
Ah, thanks @rockysui I just made a quick update to the bridgedGEPIterator repo, and looks better but I'll try to verify in more detail later this afternoon. But a preliminary look seems good:

```
[+] Compare with our bridged implementation
%struct.MyStruct*
i32 0
%struct.MyStruct = type { i32, i8* }
i32 1
```

I'll also try to look at the suggested change @dtzWill (and it seems like a good fit at first glance to me), then we might have two alternative solutions...